preprocessing/tabular_data/feature_selection_tabular_data.ipynb

```diff
@@ -0,0 +1,351 @@
+{
```

Comment:
<p>After selecting your desired approach, move onto the next cell to prune features that have a importance score less than or equal to a <code>threshold</code> value. Depending on the approach of your choice and the distribution of scores, the threshold value may vary.</p><p> </p><p>In this example, we select the first approach with XGBoost and set the <code>threshold</code> value to 0.01.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1664/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>